packages/assistify-help-request/client/views/creationDialog/AssistifyCreateRequest.js

```diff
@@ -162,6 +167,13 @@ Template.AssistifyCreateRequest.onCreated(function() {
 	instance.requestTitle = new ReactiveVar('');
 	instance.openingQuestion = new ReactiveVar('');
 
+	if (FlowRouter._current.queryParams) {
+		const expertise = FlowRouter._current.queryParams['topic'] || FlowRouter._current.queryParams['expertise'];
+		if (expertise) {
+			instance.expertise.set(expertise);
```

Comment:
Where is `instance.expertise` printed into the DOM?
Reading a HTTP request parameter and print it into the DOM, is a typical `cross site scripting` pitfall.
Did you check that the value of `instance.expertise.get()` is being escaped?

Reply:
I'd expect FlowRouter to do the escaping, but let me validate...

Reply:
`FlowRouter` moves the parameters to an object. The properties are already escaped.

http://localhost:3000/create-channel?topic=demo&title=something&question=%3Cscript%3Evar%20buh%20=%20%22buh!%22%22%3C/script%3E leads to a string in the input.